codespell_lib/data/dictionary.txt

```diff
@@ -5414,6 +5482,7 @@ detction->detection
 detecs->detects, deters, detect,
 detectected->detected
 detectetd->detected
+detectrename->detectrenames
```

Comment:
I don't think `detectrenames` is English, I  suggest to remove this entry

Reply:
Removed.